core/controllers/topics_and_skills_dashboard.py

```diff
@@ -79,6 +80,21 @@ def get(self):
                             self.user, topic_rights)
                     )
 
+        all_classroom_dict = config_domain.TOPIC_IDS_FOR_CLASSROOM_PAGES.value
+        classroom_names = [
+            classroom['name'] for classroom in all_classroom_dict]
+
+        for topic_summary_dict in topic_summary_dicts:
+            classroom_is_assigned_to_topic = False
```

Comment:
It's really topic_is_assigned_to_classroom

Reply:
Done.